obelisk_ws/src/obelisk_msgs/obelisk_sensor_msgs/msg/ObkJointEncoders.msg

```diff
@@ -0,0 +1,7 @@
+string MESSAGE_NAME="ObkJointEncoders"
+
+builtin_interfaces/Time stamp
+
+float64[] joint_pos
+float64[] joint_vel
+string[] joint_names
```

Comment:
So we require the order and length of `joint_pos` and `joint_vel` to be the same, right? I'm wondering if there's some clean way to enforce that in the message type, but I'm not really seeing it

Reply:
also, if you have a joint with a quaternion state, then the velocity joint won't be the same size.

Reply:
Yes, agree. We should at the very least document what the expected convention is for these things.